src/routes/index.js

```diff
@@ -20,72 +21,158 @@ export const configureRoutes = (app, passport) => {
   app.get('/fail', (req, res) => {
     res.send('You are not authorised');
   });
-
-  //  /api/products -  Return all products
-  //  /api/products -  Add new product and return it
+  
+  
+  /**
+   *   -----------Product's Routes-----------
+   */
+  
+  //  /api/products - GET -  Return all products
+  //  /api/products - POST - Add new product and return it
   app.route('/api/products')
-    .get(checkToken, (req, res) => {
-      Product.findAll()
-        .then(products => {
-          res.send(products);
+    .get((req, res) => {
+      Product.find()
+        .then(products => res.json(products))
+        .catch(err => {
+          res.send('Something went wrong. You can not get Products');
+          throw err;
```

Comment:
Don't throw exception here - it's a bad practice because Node app can fall down in this case.
I propose you just write it in logger and sent 500 status to client like here:
`
err => {
   logger.error('Error in database, See log: ' err.message);
  return res.sendStatus(500).json({ ...err });
}
`

Reply:
Done